### ISSUE TITLE
[BUG] Option `placeholder` with query `size` causes an error. Version 3.0.2 only.

### ISSUE THREAD

**Issue by webdiscus** [dazuaz/responsive-loader#153]
When the option `placeholder=true` and an image has the query parameter `size` causes the error:

```
ERROR in ./src/assets/images/logo.jpg?size=400
Module build failed (from ../../../node_modules/responsive-loader/lib/cjs.js):
TypeError: Cannot read properties of undefined (reading 'path')
    at transform (.../node_modules/responsive-loader/lib/index.js:134:29)
```

This error causes only with version `3.0.2`, with previous `2.3.0` was all OK.

## Reproduce
Image request: `image.jpg?size=400`

Loader options:
```js
{
  name: 'assets/img/[name]-[width]w.[ext]',
  sizes: [120, 200, 320],
  format: 'webp',
  placeholder: true, // if is false, then OK
}
```

The bug is in the `function transform()` at this line:

```js
const defaultImage = outputPlaceholder ? files[files.length - 2] : files[files.length - 1]; // here is bug
return `${esModule ? 'export default' : 'module.exports ='} {
        srcSet: ${srcset},
        images: [${images}],
        src: ${defaultImage.path},
        toString: function(){return ${defaultImage.path}},
        ${placeholder ? 'placeholder: ' + placeholder + ',' : ''}
        width: ${defaultImage.width},
        height: ${defaultImage.height}
      }`;
```

The array `files` has the size `1` then `files.length - 2` is `-1` => `defaultImage = undefined`, then `undefined.path` => ERROR

**Comment by dazuaz** [comment]
@webdiscus Thanks for pointing out the bug, it has been patched on 3.0.3.

**Comment by webdiscus** [comment]
@dazuaz Thanks a lot for the quick fix. Now it works.
